ports/geckolib/gecko_bindings/bindings.rs

```diff
@@ -251,7 +253,7 @@ extern "C" {
                                           classList: *mut *mut *mut nsIAtom)
      -> u32;
     pub fn Gecko_GetServoDeclarationBlock(element: *mut RawGeckoElement)
-     -> *mut ServoDeclarationBlock;
+     -> ServoDeclarationBlockBorrowed;
```

Comment:
And yeah, it compiles. Would you consider it as a bug of the compiler to allow this?